account/views.py

```diff
@@ -102,4 +102,29 @@ def post(self, request):
         new_access_token = str(RefreshToken(refresh_token).access_token)
         response = Response({"detail": "token refreshed"}, status=status.HTTP_200_OK)
         response.set_cookie("access_token", value=str(new_access_token), httponly=True)
-        return response
\ No newline at end of file
+        return response
+    
+class LogoutView(APIView):
+    @swagger_auto_schema(
+        operation_id="로그아웃",
+        operation_description="refresh token을 삭제합니다.",
+        request_body=LogoutRequestSerializer,
+        responses={204: "No Content", 400: "Bad Request", 401: "Unauthorized"},
+    )
```

Comment:
로그아웃을 하기 위해서는 우선 로그인이 된 상태여야겠죠!! 로그인이 되어 있는 사용자는 유효한 access token을 가지고 있는 것이 일반적입니다. 요청을 보낼 때 header에 access token을 담아서 보내도록 하면 더 좋을 것 같아요 ~!
```suggestion
    manual_parameters=[openapi.Parameter("Authorization", openapi.IN_HEADER, description="access token", type=openapi.TYPE_STRING)])
```